.eslintrc

```diff
@@ -1,14 +1,11 @@
 {
-  "extends": [
-    "eslint:recommended",
-    "plugin:promise/recommended",
-    "prettier"
-  ],
-  "plugins": [
-    "promise"
-  ],
+  "extends": ["eslint:recommended", "plugin:promise/recommended", "prettier"],
+  "plugins": ["promise"],
   "parserOptions": {
-    "ecmaVersion": 2019
+    "ecmaVersion": 8
```

Comment:
Isn't `8` equivalent to `2017`? Why the regression?

Reply:
reverted the change